SwiftWisdom/Core/Colors/ColorDescriptor.swift

```diff
@@ -64,18 +79,25 @@ extension ColorDescriptor : StringLiteralConvertible, RawRepresentable, Equatabl
         self.init(rawValue)
     }
     
-    init(_ string: String) {
+    public init(_ string: String) {
         let rgbComponents = string.componentsSeparatedByString(",")
-            .flatMap { Int($0) }
         if rgbComponents.count == 4 {
-            self = .RGB(r: rgbComponents[0], g: rgbComponents[1], b: rgbComponents[2], a: rgbComponents[3])
+            if rgbComponents.first?.containsString(".") == true {
```

Comment:
The alternative I was toying with is checking the entire string for a `.`, that might be a better approach.